### PR TITLE
Italian transl

### PR DESCRIPTION
**Pull request by ShardanaSoft**
Fix for #812 
this looks way better to me. I prefer anyway open a pr just so you can judge and fix some other possible errors.

### CODE REVIEW

File: src/js/react_views/IntlHelperBarView.jsx [hunk 1]
```diff
@@ -133,6 +133,12 @@ class IntlHelperBarView extends React.Component{
         this.fireCommand('locale ta_IN; levels');
       }.bind(this)
     }, {
+        text: "italiano",
```
Comment (pcottle) [inline src/js/react_views/IntlHelperBarView.jsx:136]: great!

**Comment by pcottle** [comment]
Much better 👏  I'll merge this for now and push the site when everything lands perhaps?

**Comment by ShardanaSoft** [comment]
yeah wait for the final push. 
just to be sure, I have to update my local repo before pushing again right? and use the same branch

**Comment by pcottle** [comment]
> just to be sure, I have to update my local repo before pushing again right? and use the same branch

yeah I would pull locally from remote master (which will show your merge commit) and then make a new PR off of that tip of master on a new branch. then itll only show the changes since this PR

**Pull request merged**